Sources/StoredSetting.swift

```diff
@@ -8,30 +8,30 @@
 
 public struct StoredSetting<T: AnyObject> {
   private let key: String
-  private let settings: NSUserDefaults
+  private let settings: UserDefaults
```

Comment:
fyi most of these were converted to fileprivate -- but i guessed since they weren't using extensions private was fine.